packages/create-analog/template-blog/src/app/app.config.ts

```diff
@@ -0,0 +1,14 @@
+import { provideHttpClient } from '@angular/common/http';
+import { ApplicationConfig } from '@angular/core';
+import { provideClientHydration } from '@angular/platform-browser';
+import { provideFileRouter } from '@analogjs/router';
+import { provideContent, withMarkdownRenderer } from '@analogjs/content';
+
+export const appConfig: ApplicationConfig = {
+  providers: [
+    provideFileRouter(),
+    provideHttpClient(),
```

Comment:
Let's default to withFetch also to get rid of the console warning.

```suggestion
    provideHttpClient(withFetch()),
```

Reply:
There's something new I learned 😄